### ISSUE TITLE
Bug in Indentation for Return

### ISSUE THREAD

**Issue by rmorshea** [DSpeckhals/python-indent#47]
Improper indentation occurs when entering an empty hanging indent on a line which contains `return`:

*"|" is used as cursor placement.*

---

**Actual**

```python
def func():
    return (
    |
)
```

**Expected**

```python
def func():
    return (
        |
    )
```

---

Everything works as expected when the contents are not empty:

```python
def func():
    return (1, 2, 3,
            |)
```

**Comment by kbrose** [comment]
I'm guessing this will be the first of many issues that come from the changes to the built-in python indenting. Hopefully I'll have some time to dig into this tomorrow. If only the Atom team would actually build useful APIs for indentation we could avoid this altogether...

**Comment by DSpeckhals** [comment]
Yes. I was wondering about that too. The nice thing is that the tests _should_ catch most problems through the beta channel before they affect all users.

**Comment by DSpeckhals** [comment]
Fixed in #48. I just published version 1.1.1 of this package.
